src/components/navMenu/navMenu.tsx

```diff
@@ -19,6 +19,7 @@ const NavMenu = ({ render }) => {
         },
         ":hover": {
```

Comment:
Btw, the Chakra convention is `_hover`. This does the same thing and is the standard CSS convention.